lib/core/mount.js

```diff
@@ -168,9 +169,13 @@ function mount(mountPath, parentApp, events) {
 
 		// Initialize the session store
 		try {
-			
-			var SessionStore = require(sessionStore)(express);
-			
+      if(sessionStore === 'connect-redis') {
+				expressSession = require('express-session');
```

Comment:
`express-session` is available above on https://github.com/keystonejs/keystone/blob/master/lib/core/mount.js#L57